src/ImmersedBoundaries/mask_immersed_field.jl

```diff
@@ -6,9 +6,35 @@ using Oceananigans.Fields: location, ZReducedField, Field
 
 instantiate(X) = X()
 
+#####
+##### Outer functions
+#####
+
+function mask_immersed_field!(field::Field, value=zero(field.grid); blocking=true)
+    if blocking
+        event = mask_immersed_field!(field, field.grid, location(field), value)
+        wait(device(architecture(field)), event)
+        return nothing
+    else
+        return mask_immersed_field!(field, field.grid, location(field), value)
+    end
+end
+
+function mask_immersed_reduced_field_xy!(field::ZReducedField, value=zero(field.grid); k, blocking)
```

Comment:
```suggestion
function mask_immersed_reduced_field_xy!(field::ZReducedField, value=zero(field.grid); k, blocking)
```
function mask_immersed_field_xy!(field value=zero(field.grid); k, blocking, mask=peripheral_node)